client/my-sites/plugins/marketplace/components/index.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { HorizontalRule } from '@wordpress/components';
+import styled from '@emotion/styled';
+import { MarketplaceThemeType } from 'calypso/my-sites/plugins/marketplace';
```

Comment:
Nit: While we're still deciding whether to keep those in #52499, we generally still use the `External dependencies` and `Internal dependencies` comment blocks and put the corresponding imports where they belong.